docs/cli.html

```diff
@@ -948,7 +948,7 @@ <h2 id="logs">logs</h2>
 <li><code>--status={any,success,error}</code> -- <em>optional</em>, display only success logs (status code &lt; 400 / info) or error (status code &gt; 400 / tracebacks). Default is <code>any</code></li>
 <li><code>--type={console,http}</code> -- <em>optional</em>, display only console or http logs. Default is <code>console</code></li>
 <li><code>--detailed</code> -- <em>optional</em>, show detailed logs (like request/response body and headers)</li>
-<li><code>--user=user@example.com</code> -- <em>optional</em>, display only this user&apos;s logs. Default is <code>me</code></li>
+<li><a href="mailto:`--user=user@example.com">`--user=user@example.com</a><code>-- _optional_, display only this user&apos;s logs. Default is</code>me`</li>
```

Comment:
not sure! I didn't change this, but it popped out in the docs. I just did a `rm -rf node_modules && npm i && npm run docs` and had no local changes on this branch. the `package-lock` should guarantee that others have the same result.

Reply:
No worries, was just wondering.

Reply:
so I dug in here. seems like it's because litdoc (which generates docs) installs a range and last time the package-lock got cleared, we slid up a version that changed the way they parsed emails ([this](https://github.com/markedjs/marked/pull/1034) PR, probably went out in `0.3.14`). I agree it looks a little weird, but probably not worth fixing at this point